### PR TITLE
setting random comleted

### PR DESCRIPTION
**Pull request by RaLLy08**

### CODE REVIEW

File: src/Controller.js [hunk 1]
```diff
@@ -10,6 +10,16 @@ class Controller {
     init = () => {
         this._view.init();
     }
+
+    setRandomConfig = (value, minAge, maxAge, minHealth, maxHealth, minMoney, maxMoney, minOffenses, maxOffenses) => {
```
Comment (hnamazli) [inline src/Controller.js]: wou wou wou, this is not correct for arguments.
can use an object?

File: src/Controller.js [hunk 2]
```diff
@@ -10,6 +10,17 @@ class Controller {
...
+        for (let i = 0; i <= value; index++) {
+            age = Math.floor(Math.random() * (maxAge - minAge) + minAge);
+            health = Math.floor(Math.random() * (maxHealth - minHealth) + minHealth);
+            money = Math.floor(Math.random() * (maxMoney - minMoney) + minMoney);
+            offenses = Math.floor(Math.random() * (maxOffenses - minOffenses) + minOffenses);
+            this.createPerson(age, health, money, offenses);
```
Comment (hnamazli) [inline src/Controller.js]: code style

**Review by hnamazli — COMMENTED** [review]
Please use the correct arguments for this functions

File: src/Controller.js [hunk 3]
```diff
@@ -12,14 +12,15 @@ class Controller {
     }
 
     setRandomConfig = (value, minAge, maxAge, minHealth, maxHealth, minMoney, maxMoney, minOffenses, maxOffenses) => {
-        for (let i = 0; i >= value; index++) {
+        for (let i = 0; i <= value; index++) {
             age = Math.floor(Math.random() * (maxAge - minAge) + minAge);
             health = Math.floor(Math.random() * (maxHealth - minHealth) + minHealth);
             money = Math.floor(Math.random() * (maxMoney - minMoney) + minMoney);
             offenses = Math.floor(Math.random() * (maxOffenses - minOffenses) + minOffenses);
             this.createPerson(age, health, money, offenses);
```
Comment (hnamazli) [inline src/Controller.js]: code style

**Review by hnamazli — APPROVED** [review]

**Review by Musa-94 — APPROVED** [review]

**Pull request merged**